java/src/org/certificatetransparency/ctlog/LogEntry.java

```diff
@@ -0,0 +1,12 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class LogEntry {
+  public Ct.X509ChainEntry x509Entry;
+  public Ct.PrecertChainEntry preCertEntry;
```

Comment:
Oh, I remember, I was looking at the local variable that it was being set from. Changed it.

Of note, this field is write-only. :stuck_out_tongue: